tools/flye/flye.xml

```diff
@@ -91,9 +100,10 @@
         <test>
             <param name="inputs" ftype="fasta" value="nanopore.fasta"/>
             <param name="mode" value="--nano-corr"/>
-            <param name="g" value="10000"/>
             <param name="i" value="2"/>
-            <param name="asm" value="40"/>
+            <param name="asm|asm_select" value="true" />
```

Comment:
can you please use the `<conditional name="">` syntax?

Reply:
oh, sure, can do.